index.html

```diff
@@ -0,0 +1,73 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width , initial-scale=1">
+    <meta http-equiv="X-UA-Compatible" content="ie=edge">
+    <title>KMarketer: We Take Care of the Future</title>
+    <link rel="stylesheet" href="./css/style.css">
+</head>
+
+<body>
+    <form action="/action_page.php" method="get">
+        <div class="introduction"> 
+    <header class="header">
+        <div class="wrapper">
+            <div class="header_wrapper">
+                <div class="header_logo">
+                    <a href="/" class="header_logo-link">
+                        <img src="./img/svg/KMarketer_Logo.svg" alt="We Take Care of the Future"
+                            class="header_logo-pic">
+                    </a>
+                </div>
+
+                <div>
+                    <nav class="header_nav">
+                        <ul class="header_list">
+                            <li class="header_item">
+                                <a href="#!" class="header_link">Home</a>
+                            </li>
+                            <li class="header_item">
+                                <a href="#!" class="header_link">Product</a>
+                            </li>
+                            <li class="header_item">
+                                <a href="#!" class="header_link">Pricing</a>
+                            </li>
+                            <li class="header_item">
+                                <a href="#!" class="header_link">Contact</a>
+                            </li>
+                            <ul class="header_icons">
+                                <li>
+                                    <a href="#!" class="header_logo-link">
+                                        <img src="./img/svg/nav_bar_search_icon.svg" alt="search_icon"
+                                            class="header_icons">
+                                    </a>
+                                </li>
+                                <li>
+                                    <a href="#!" class="header_logo-link">
+                                        <img src="./img/svg/nav_bar_shopping_cart_icon.svg" alt="shoppig_cart_icon"
+                                            class="header_icons">
+                                    </a>
+                                </li>
+                            </ul>
+                        </div>
+    </header>
+
+        <div class="main-container">
+    <h1>We Take Care of <br> the Future 
+</h1>
+<h4>We know how large objects will act, but things on a <br> 
+    small scale just do not act that way.</h4>
+
+<div class="buttons">
+    <button class="button-get-quote">Get Quote Now</button>
+    <button class="button-learn-more">Learn More</button>
+</div>
+</div>
+</div>
```

Comment:
these closing tags are not on their places. 
Remember the opening and closing tags should be always pared: `<div> <h1></h1></div>`